.tekton/osc-caa-push.yaml

```diff
@@ -26,6 +26,8 @@ spec:
     value: cloud-api-adaptor/Dockerfile.openshift
   - name: path-context
     value: src
+  - name: prefetch-input
+    value: '[{"type": "rpm", "path": "./src/cloud-api-adaptor/"}, {"type": "gomod", "path": "./src/cloud-api-adaptor"}]'
```

Comment:
Like the above

```suggestion
value:
  - type: rpm
    path: ./src/cloud-api-adaptor/
  - type: gomod
    path: ./src/cloud-api-adaptor
```

Reply:
Resolving this - let's discuss it on the other one